Testreglar/2.3.1/Nett/nett-2.3.1a.json

```diff
@@ -0,0 +1,212 @@
+{
+	"namn": "Nett-2.3.1a Nettsida har ikkje innhald som glimtar",
+	"id": "nett-2.3.1a",
+	"testlabId": 521,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nn",
+	"kravTilSamsvar": "<ul><li>Nettsiden har ikke innhold som glimter eller</li><li>Nettsiden har innhold som glimter, og en av følgende oppfylt:<ul><li>Innholdet dekker mindre enn 21 824 kvadratpiksler (25% av skjermen)</li><li>Det er færre enn tre blink i sekundet</li></ul></li></ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Kva side testar du?",
+			"ht": "<p>Angi URL eller side-ID</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsida innhald som glimtar?",
+			"ht": "<p>Du skal vurdere alt innhald på nettsida. Dette omfattar også bilde, animasjonar og videoklipp.</p><p>Slik testar du:</p><ul><li>Gjer ein visuell inspeksjon og sjå etter innhald som glimtar<ul><li>Glimting er veksling mellom to visuelle tilstandar for å vekke merksemd.</li><li>Glimt er motsatte parvise skift i relativ lysstyrke som kan forårsake (epileptiske) anfall.</li></ul></li><li>Sørg for at nettsida har lasta inn fullstendig, før du går i gang med testen.</li></ul><p><strong>Merk:</strong></p><ul><li>Glimting på grunn av skjerm eller innlasting av nettsida skal ikkje testast.</li><li>Du skal ikkje teste videoklipp i fullskjermsvisning.</li></ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Testside har ikkje innhald som glimtar."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Sett oppløysinga på skjermen du brukar til 1024*768.",
```

Comment:
Skal vi ikke bruke samme oppløsning som ved 1.4.4. Det er nye og oppdaterte tall som er bedre for 2024.